helm-chart/ray-cluster/values.yaml

```
@@ -16,6 +16,30 @@ fullnameOverride: ""
 imagePullSecrets: []
   # - name: an-existing-secret
 
+# gcsFaultTolerance specifies configuration for GCS fault tolerance.
+# See https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/kuberay-gcs-ft.html#kuberay-gcs-ft
+# for more details.
+gcsFaultTolerance:
+  enabled: false
+  # In most cases, you don't need to set `externalStorageNamespace` because KubeRay will
+  # automatically set it to the UID of RayCluster. Only modify this annotation if you fully understand
+  # the behaviors of the Ray GCS FT and RayService to avoid misconfiguration.
+  # externalStorageNamespace: "my-raycluster-storage"
+  # redisAddress: "redis:6379"
+  # redisPassword:
+    # value: ""
+    # valueFrom: {}
+    #   secretKeyRef:
```

```suggestion
    # valueFrom:
    #   secretKeyRef:
```
Please mention that the user should create the secret first.

---

Fixed in https://github.com/ray-project/kuberay/pull/3881/commits/31574d7bdb636e850e9a4ed200265f6986a33b13.